Standalone/PythonTests/Automated/benchmark_runner_periodic_suite.py

```diff
@@ -0,0 +1,56 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+import logging
+import os
+import subprocess
+
+import pytest
+
+import ly_test_tools.environment.process_utils as process_utils
+import ly_test_tools.launchers.platforms.base
+from .benchmark_data_aggregator import BenchmarkDataAggregator
+
+logger = logging.getLogger(__name__)
+
+
+class AtomSampleViewerException(Exception):
+    """Custom Exception class for AtomSampleViewer tests."""
+    pass
+
+
+@pytest.mark.parametrize('launcher_platform', ['windows'])
+@pytest.mark.parametrize("project", ["AtomSampleViewer"])
+@pytest.mark.parametrize('rhi', ['dx12', 'vulkan'])
+@pytest.mark.usefixtures("clean_atomsampleviewer_logs", "atomsampleviewer_log_monitor")
+class TestPerformanceBenchmarksPeriodicSuite:
+    def test_BerformanceBenchmarkPeriodicSuite(self, request, workspace, launcher_platform, rhi, atomsampleviewer_log_monitor):
+        print("FDKSJHFDSHFSDKF")
```

Comment:
what is this for? can we remove it?

Reply:
Oops, debugging statement I forgot to remove.